NewKernel_d/include/CGAL/NewKernel_d/Coaffine.h

```diff
@@ -20,6 +20,13 @@
 
 #ifndef CGAL_KD_COAFFINE_H
 #define CGAL_KD_COAFFINE_H
+
+#if defined(BOOST_MSVC)
+#  pragma warning(push)
+#  pragma warning(disable:4913) // user defined binary operator ',' exists but no overload could convert all operands,
+                                // default built-in binary operator ',' used
+#endif
```

Comment:
You might want to discuss with @mglisse why the warning [C4913] has to build quieted. That is really strange. Maybe the overloaded `operator,` is from Eigen, but in that case maybe Eigen developers should know about it.

[C4913]: https://docs.microsoft.com/en-us/cpp/error-messages/compiler-warnings/compiler-warning-level-4-c4913

Reply:
Disabling this warning seems like the right thing. Just because there is one `operator,` somewhere in the code base, all other normal uses of `,` now get a warning??? Nothing Eigen can do I think, I am using `,` in my code.

Reply:
You are right. C4913 is just another stupid verbose warning of `/w4`.

Reply:
We could #include <CGAL/push.h> and  <CGAL/pop.h> which suppress warnings. That is maybe better than what I currently add on an *as-needed* base.  Fact is that I like some warnings such as for unreachable code.